<?xml version="1.0"?>

<launch>

    <arg name="rosbag" default="/home/mark/rosBags/June_11/test3.bag"/>
    <arg name="rosbag_play_rate" default="20"/>
    <arg name="filename" default="output3.avi"/>


  <node pkg="rosbag" type="play" name="rosbag" required="true" args="$(arg rosbag) --rate $(arg rosbag_play_rate)"/>

 <node name="video_extraction_node" pkg="video_extraction" type="video_extraction_node" output="screen">
   <remap from="camera/image_raw" to="tracks_video/image_raw"/>
   <param name="transport" value="compressed"/>
   <param name="resize" value="true"/>
   <param name="resize_width" value="640"/>
   <param name="resize_height" value="480"/>
   <param name="fps" value="5"/>
   <param name="fourcc" value="H264"/>
   <param name="filename" value="$(arg filename)"/>
 </node>

<!--  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
 </node> -->
 
 
</launch>